src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's address in the address book.
+ * Serves as a parent class for other classes
```

Comment:
This line can be omitted. Any class can serve as a parent class for other classes (unless it is declared as `final`)